pkg/driver/failure/triggers.go

```diff
@@ -0,0 +1,108 @@
+package failure
+
+import (
+	"github.com/sirupsen/logrus"
+	"github.com/vhive-serverless/loader/pkg/config"
+	"os/exec"
+	"strings"
+	"sync"
+	"time"
+)
+
+const NodeSeparator = " "
+
+func ScheduleFailure(platform string, config *config.FailureConfiguration) {
+	if config != nil && config.FailAt != 0 && config.FailComponent != "" {
+		switch platform {
+		case "Knative", "Knative-RPS":
+			triggerKnativeFailure(config.FailNode, config.FailComponent, config.FailAt)
+		case "Dirigent", "Dirigent-RPS":
+			triggerDirigentFailure(config.FailNode, config.FailComponent, config.FailAt)
+		default:
+			logrus.Errorf("No specified failure handler for given type of system.")
+		}
+	}
+}
+
+func triggerKnativeFailure(nodes string, component string, t int) {
+	time.Sleep(time.Duration(t) * time.Second)
```

Comment:
I would recommend moving this line into `ScheduleFailure`. This would deduplicate it and make it more obvious.